nitro-protocol/src/bitfield-utils.ts

```diff
@@ -29,10 +29,18 @@ export function getSignersIndices(signedByStr: Uint256): number[] {
   return signerIndices;
 }
 
-export function getSignedBy(signerIndices: number[]): Uint256 {
+export const SIGNED_BY_NO_ONE = 0;
+
+export function getSignedBy(signerIndices: number | number[]): Uint256 {
```

Comment:
This change should probably have some added test cases in `bitfield-utils.test.ts` as well.

Reply:
Implemented here [af80245](https://github.com/statechannels/go-nitro/pull/1044/commits/af8024559a544edcf7db1a8c6ebb444cbc707320).